libraries/SPISlave/src/SPISlave.cpp

```diff
@@ -64,13 +64,13 @@ void SPISlaveClass::_s_status_tx(void *arg)
     reinterpret_cast<SPISlaveClass*>(arg)->_status_tx();
 }
 
-void SPISlaveClass::begin()
+void SPISlaveClass::begin(uint8_t statusLength)
```

Comment:
Done. Why it's so important to follow current signature? This default value will not break compatibility, I think.

Reply:
For callers, a change of signature shouldn't be a problem.
For derived classes and some template cases, it can be a compiler error or a silent bug even.